client/src/main/java/com/linecorp/decaton/client/internal/DecatonClientImpl.java

```diff
@@ -81,6 +82,19 @@ public CompletableFuture<PutTaskResult> put(String key, T task, TaskMetadata ove
         return put(key, task, convertToTaskMetadataProto(overrideTaskMetadata));
     }
 
+    @Override
+    public CompletableFuture<PutTaskResult> put(String key, T task, TaskMetadata overrideTaskMetadata,
```

Comment:
This method supersedes below `put(String key, T task, TaskMetadataProto taskMetadataProto)` so let's change it to just calling this method?